website/docs/reference/configs-and-properties.md

```diff
@@ -26,9 +26,18 @@ Whereas you can use **configurations** to:
 
 Depending on the resource type, configurations can be defined in the dbt project and also in an installed package by:
 
+<VersionBlock firstVersion="1.9">
+
+1. Using a [`config` property](/reference/resource-properties/config) in a `.yml` file in the `models/`, `snapshots/`, or `tests/` directory
+2. From the [`dbt_project.yml` file](dbt_project.yml), under the corresponding resource key (`models:`, `snapshots:`, `tests:`, etc)
+</VersionBlock>
+
+<VersionBlock lastVersion="1.8">
+
 1. Using a [`config()` Jinja macro](/reference/dbt-jinja-functions/config) within a `model`, `snapshot`, or `test` SQL file
-2. Using a [`config` property](/reference/resource-properties/config) in a `.yml` file
+2. Using a [`config` property](/reference/resource-properties/config) in a `.yml` file in the `models/`, `snapshots/`, or `tests/` directory.
```

Comment:
@dbeatty10 can i gut check this with you if this is right?

Reply:
I think it's going to be _very close_ to the **default** list of folders in the [`docs-path`](https://docs.getdbt.com/reference/project-configs/docs-paths), namely:
- [`model-paths`](https://docs.getdbt.com/reference/project-configs/model-paths)
- [`seed-paths`](https://docs.getdbt.com/reference/project-configs/seed-paths)
- [`analysis-paths`](https://docs.getdbt.com/reference/project-configs/analysis-paths) \*
- [`test-paths`](https://docs.getdbt.com/reference/project-configs/test-paths)
- [`snapshot-paths`](https://docs.getdbt.com/reference/project-configs/snapshot-paths)

But not:
- [macro-paths](https://docs.getdbt.com/reference/project-configs/macro-paths)

(\* I'd need to double-check `analysis-paths` to see if it supports `config:` or not.)

Reply:
I double-checked, and the analyses within the `analysis-paths` directory **do** support some configs (like `tags`, `enabled`, and `meta`).

Reply:
So the changes here under the `.yml` file items are missing `seeds` and `analyses`.